pycobertura/reporters.py

```diff
@@ -95,6 +95,9 @@ def generate(self):
 
 class HtmlReporter(TextReporter):
     def __init__(self, *args, **kwargs):
+        self.title = kwargs.pop("title", "pycobertura report")
+        self.inline_sources = kwargs.pop("inline_sources", True)
```

Comment:
Can we rename this to:

```suggestion
        self.render_file_sources = kwargs.pop("render_file_sources", True)
```

...?